bedrock/newsletter/templates/newsletter/existing.html

```diff
@@ -52,7 +52,7 @@
   {% endblock %}
 
   {% if formset and not switch('newsletter-maintenance-mode') %}
-    <form method="post" action="{{ secure_url() }}" id="existing-newsletter-form" class="container billboard"
+    <form method="post" action="{{ request.build_absolute_uri(request.path) }}" id="existing-newsletter-form" class="container billboard"
```

Comment:
This may actually be easier to leave as-is, as the `token` isn't currently sent into the template, and there are may `render` calls in the view. Will add a comment though.